examples/player_info/reds/DumpPlayerInfo.reds

```diff
@@ -0,0 +1 @@
+native func DumpPlayerInfo(player: ref<PlayerPuppet>) -> Void;
```

Comment:
This file is unnecessary too, I'd remove it to keep things simple

Reply:
done in f724dd4